templates/archive.html

```diff
@@ -5,18 +5,20 @@
 	<!-- wp:query-title {"type":"archive","style":{"spacing":{"margin":{"bottom":"100px"}}}} /-->
 
 	<!-- wp:post-template -->
-		<!-- wp:group -->
-		<div class="wp-block-group">
+		<!-- wp:group {"tagName":"article"} -->
+		<article class="wp-block-group">
```

Comment:
Since this is repeated for each post; -for each individual result on the archive page, won't this create too many landmarks?